app/controllers/course/enrol_requests_controller.rb

```diff
@@ -0,0 +1,127 @@
+class Course::EnrolRequestsController < Course::ModuleController
+  load_and_authorize_resource through: :course
+  skip_before_filter :verify_authenticity_token, only: [:approve_selected, :delete_selected]
+
+  def index #:nodoc:
+    @staff_requests = @enrol_requests.staff
+    @student_requests = @enrol_requests.student
+
+    respond_to do |format|
+      format.html
+    end
+  end
+
+  def new #:nodoc:
+    unless current_user
+      redirect_to new_user_session_path
+      return
+    end
+    unless @course.opened?
+      redirect_to course_path(@course), flash: {error: t('.course_not_open')}
+      return
+    end
+
+    existing = Course::EnrolRequest.find_by_user_id_and_course_id(current_user.id, @course.id)
+    if existing
```

Comment:
Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.